### PR TITLE
Hyperledger AnonCreds Q3 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Framework JavaScript implementation supports multiple AnonCreds publication
+platforms beyond Hyperledger Indy, including at least [Cheqd.io] and [Cardano].
+
+[AnonCreds Rust]: https://github.com/hyperledger/anoncreds-rs
+[Aries Framework JavaScript]: https://github.com/hyperledger/aries-framework-javascript
+[Aries Bifold]: https://github.com/hyperledger/aries-mobile-agent-react-native
+[Cheqd.io]: https://cheqd.io/
+[Cardano]: https://cardano.org/
+
+Work on embedded the [AnonCreds Rust] implementation in [Aries Cloud Agent Python]
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md]:
```suggestion
Work on embedding the [AnonCreds Rust] implementation in [Aries Cloud Agent Python]
```

**Review by tkuhrt — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]

File: project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+- [Hyperledger AnonCreds Wiki Home Page]
+
+[Hyperledger AnonCreds Website Home Page]: https://www.hyperledger.org/use/anoncreds
+[Hyperledger AnonCreds Wiki Home Page]: https://wiki.hyperledger.org/display/ANONCREDS/
+
+## Project Health
+
+The project continues to make good progress, with the maintainers and
+contributors completing version 0.1.0 of the AnonCreds Rust implementation and
+that library being used in downstream applications.
```
Comment (jimthematrix) [inline project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md:47]: "downstream applications" sounds vague, could use a clarification on whether that's "applications built by adopters of AnonCreds", or "applications in the AnonCreds project", or "applications built by other teams in the foundation"?

**Review by jimthematrix — APPROVED** [review]
approved with a minor question for clarification

**Review by bobbi28 — APPROVED** [review]

**Review by petermetz — APPROVED** [review]
LGTM

**Review by denyeart — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md:47]: Clarified this — adding `, including the various Aries components/frameworks, and applications built on Aries.`

**Review by lehors — APPROVED** [review]

**Pull request merged**